src/Minigames/dungeon.js

```diff
@@ -16,8 +16,9 @@ class Entity {
 
     fight(Entity, damage) {
         let dmg = Math.floor(Math.max(damage - Entity.defense, 1))
+        let old = Entity.hp
         Entity.hp = Entity.hp - dmg
-        return (this.ai ? "The " + this.ai + " attacks you!" : "You attack " + Entity.ai + "!") + " (" + dmg + " damage dealt)"
+        return (this.ai ? "The " + this.ai + " attacks you!" : "You attack " + Entity.ai + "!") + " (" + Math.max(old, dmg) + " damage dealt)"
```

Comment:
this only changes the message string that appears in the Embed, the actual hp of the enemy is not altered and will still go into the negatives